src/cli/cli.cpp

```diff
@@ -1281,6 +1284,35 @@ void Interpreter::ProcessLinkQuality(int argc, char *argv[])
     AppendResult(error);
 }
 
+#if OPENTHREAD_FTD
+void Interpreter::ProcessPSKc(int argc, char *argv[])
+{
+    ThreadError error = kThreadError_None;
+
+    if (argc == 0)
+    {
+        const uint8_t *pskc = otThreadGetPSKc(mInstance);
+
+        for (int i = 0; i < OT_PSKC_MAX_SIZE; i++)
+        {
+            sServer->OutputFormat("%02x", pskc[i]);
+        }
+
+        sServer->OutputFormat("\r\n");
+    }
+    else
+    {
+        uint8_t pskc[OT_MASTER_KEY_SIZE];
```

Comment:
@aeliot is suggesting that the two instances be named differently to better indicate:

1. they have different types and,
2. their intended purpose.

For example, the `currentPskC` and `newPskC` or something similar.